vela-templates/definitions/internal/health.cue

```diff
@@ -0,0 +1,39 @@
+"health": {
+	annotations: {}
+	attributes: {}
+	description: "Apply periodical health checking to the application."
+	labels: {}
+	attributes: manageHealthCheck: true
+	type: "policy"
+}
+
+template: {
+	output: {
+		apiVersion: "core.oam.dev/v1alpha2"
+		kind:       "HealthScope"
+		spec: {
+			"probe-timeout":  parameter.probeTimeout
```

Comment:
How about renaming `probe-timeout` to `probeTimeout`?

Reply:
I also want to rename this field...(it's weired) but i'm not sure it's a good idea to rename CRD field name now.